HHS/Head-Start-TTADP
# [TTAHUB-2519] Add FEI root causes to report review

## Description of change

Updated the Goals and Objectives review section to show FEI root causes.

## How to test

- Review the code.
- Make sure that FEI root causes are shown on the review page.
     - Recipients that share the same causes for that goal are rolled up
     - Recipients missing root causes show the icon and 'Missing information' message.
- Make sure it shows for multiple goals on the same report.
- Make sure existing non-FEI reports still show correctly (hiding the root cause).


## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2519


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
